docker-compose.override.yaml

```
@@ -0,0 +1,13 @@
+
+services:
+  web:
+    build:
+      target: development
+    volumes:
+      - .:/app
+  bot:
+    build:
+      target: development
+    volumes:
+      - .:/app
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)